README.md

```diff
@@ -235,6 +244,41 @@ the server settings as described in the "configure dirtsand" step.
    dirtsand in a detachable GNU screen session.
 
 
+DOCKERSAND
+-------------------
+
+Docker allows building and running isolated software programs (or "containers")
+in a platform-agnostic way. This allows users to bootstrap a new DIRTSAND 
+testing server much more quickly and easily on Windows, Mac, or Linux.
+
+1) Download and install [Docker Desktop](https://www.docker.com/get-started/). If
+   on Windows, you probably want to install WSL as directed by the installer. Your
+   computer may need to restart several times during these installs.
+
+2) Clone the dirtsand repo to a local directory.
+    - NOTE: Please ensure you are using `git config core.autocrlf false` if
+    on Windows. Otherwise, the bash script files can get corrupted with the wrong
+    line endings and your server container will fail to start.
+
+3) From the local repo root directory, run `./dockersand build`.
+    - NOTE: By default, dockersand will try to use the current directory name as
+    the base docker container name. You can overwrite this usage by setting the
+    $ProjectName variable in `dockersand.ps1` (for Windows) or the PROJECT_NAME
+    variable in `dockersand.sh` (for *nix) to the desired base container name.
+
+4) Once the build is complete, run `./dockersand start` to start the new containers
+for the database and the dirtsand server. Once this command has been run, you should
+be able to see the containers in Docker Desktop and start, stop, or remove the
+containers as desired. You can also view the docker logs for the containers.
```

Comment:
```suggestion
be able to see the containers in Docker Desktop. You can also view the docker
logs for the containers.
```

The dockersand scripts were written for this purpose 😉

Reply:
Well, fair, I'll take your suggestion. I've found that starting and stopping through Docker Desktop still works as an alternative.... Just for personal edification, is there a reason I *shouldn't* use Docker Desktop for those actions?